src/analysis/rule_extraction.jl

```diff
@@ -0,0 +1,88 @@
+using SIRUS, MLJ
+using StableRNGs: StableRNG
+import ..performance: temporal_variability
+
+"""
+    Rule{V<:Vector{Float64},W<:Vector{Vector{Any}}}
+
+# A Rule contains a condition (vector of conditional clauses) and consequent (vector 
+of values for then and else results of the condition)
+"""
+struct Rule{V<:Vector{Float64},W<:Vector{Vector{Any}}}
+    condition::W
+    consequent::V
+end
+
+"""
+    rules(rules::SIRUS.StableRules)
+
+Vector of Rule objects. These are **not** the same as SIRUS.Rule object.
+
+# Arguments
+- `rules` - SIRUS.StableRules object
+
+# Return
+Vector of Rule objects
+"""
+function rules(rules::SIRUS.StableRules{Float64})
+    [Rule(_condition(rules, i), _consequent(rules, i)) for i in eachindex(rules.rules)]
+end
+
+"""
+    _condition(rules::SIRUS.StableRules{Float64}, index::Int64)
+
+Vector containing condition clauses. Each condition clause is a vector with three 
+components: a feature_name::String, a direction::String (< or ≤) and a value:<Float64 
+
+# Arguments
+- rules - SIRUS.StableRules object containing all rules
```

Comment:
Use colon instead of dash. This goes for all docstrings.

```suggestion
- rules : SIRUS.StableRules object containing all rules
```